dotnetcore3.1/cookiecutter-aws-sam-quick-start-web-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+
+namespace {{cookiecutter.project_name}}.Controllers
+{
+    [Route("api/[controller]")]
+    public class ValuesController : ControllerBase
```

Comment:
This controller class seems redundant (coming from project init maybe?). Can we remove this?

Reply:
@mndeveci I'm thinking about keeping it to show folks that they can have other Controller. Does it make sense?

Reply:
I see your point, but these are same as the ones in `BooksController` class right? In that case, I will add some documentation into README file that they can check example route configuration in `BooksController` class rather than adding this file.